Translations/translation_NL.json

```diff
@@ -40,18 +40,9 @@
 			"No USB-PD IC",
 			"detected!"
 		],
-		"LockingKeysString": [
-			" GEBLOKKEERD",
-			""
-		],
-		"UnlockingKeysString": [
-			"GEDEBLOKKEERD",
-			""
-		],
-		"WarningKeysLockedString": [
-			"!GEBLOKKEERD!",
-			""
-		]
+		"LockingKeysString": " GEBLOKKEERD",
+		"UnlockingKeysString": "GEDEBLOKKEERD",
+		"WarningKeysLockedString": "!GEBLOKKEERD!",

```

Comment:
You have a trailing comma here which is not allowed in JSON.
This will break the build for this language

Reply:
Thank you sir, will fix this. 😅